src/main/java/seedu/taassist/model/TaAssist.java

```diff
@@ -94,6 +107,32 @@ public void removeStudent(Student key) {
         students.remove(key);
     }
 
+    //// class-level operations
+
+    /**
+     * Returns true if a class with the same identity as {@code moduleClass} exists in TA-Assist.
+     */
+    public boolean hasModuleClass(ModuleClass moduleClass) {
+        requireNonNull(moduleClass);
+        return moduleClasses.contains(moduleClass);
+    }
+
+    /**
+     * Adds a class to TA-Assist.
+     * The class must not already exist in TA-Assist.
+     */
+    public void addModuleClass(ModuleClass c) {
+        moduleClasses.add(c);
+    }
```

Comment:
Maybe add `requireNonNull(c)` here to fail early rather than fail at `add(c)`

Reply:
@Bubbl3T I think it is good to have them all. Because in future we could be calling these from a new class we created.